.github/workflows/ci-additional.yml

```diff
@@ -74,6 +74,30 @@ jobs:
         shell: bash -l {0}
         run: pytest -v --doctest-modules --ignore-glob='*/test_*.py' dask
 
+  doctest-pip:
```

Comment:
It looks like we now have two doctest builds. I have a slight preference for standardizing all our CI environments to use the conda env files we maintain. But mostly I'd prefer that we only have a single doctest build